lib/faulty/circuit.rb

```diff
@@ -156,6 +156,13 @@ def finalize
       end
     end
 
+    # @return [String] Text representation of the circuit
+    def inspect
+      interested_opts = %i(cool_down evaluation_window rate_threshold sample_threshold)
+      options_text = @given_options.each_pair.map { |k,v| "#{k}: #{v}" if interested_opts.include?(k) }.compact.join(", ")
```

Comment:
We will most likely want to use the `#options` method here since those will be the ones actually used when the circuit is run. Those can differ from the given options since the backend can override them.

Reply:
Nice I can reuse the options 👍🏼